### PR TITLE
Remove remnants from "LSIF Typed"

### PR DESCRIPTION
**Pull request by olafurpg**
The codebase had a lot of outdated references to "LSIF Typed", which has been renamed to SCIP. This commit replaces these outdated references to avoid confusion for people who are reading the codebase for the first time.

### Test plan

See the CI go green. This should be a pure refactoring, no functional difference.

### CODE REVIEW

**Comment by olafurpg** [comment]
cc/ @mrnugget @valerybugakov who both asked about the usage of LSIF in scip-typescript

File: src/Descriptor.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+import * as scip from './scip'
 
-type Descriptor = lsif.lib.codeintel.lsiftyped.Descriptor
-const Descriptor = lsif.lib.codeintel.lsiftyped.Descriptor
-type Suffix = lsif.lib.codeintel.lsiftyped.Descriptor.Suffix
-const Suffix = lsif.lib.codeintel.lsiftyped.Descriptor.Suffix
+type Descriptor = scip.scip.Descriptor
+const Descriptor = scip.scip.Descriptor
+type Suffix = scip.scip.Descriptor.Suffix
+const Suffix = scip.scip.Descriptor.Suffix
```
Comment (varungandhi-src) [inline src/Descriptor.ts:6]: nit: Can we somehow avoid this being `scip.scip`. It's not a big deal, but it looks a little gross lol.

**Review by olafurpg — COMMENTED** [review]
We still copy-paste scip.ts from the sourcegraph/scip repo. It would be nice to publish the bindings to npm so that we can add a normal package.json dependency instead. We can do that later, however, this PR is already a big improvement.

File: src/Descriptor.ts (hunk 1 above)
Comment (olafurpg) [reply src/Descriptor.ts:6]: I agree it's not nice and I'd +1 if we can solve it somehow. It's not a regression from this PR though, it was gross before as well

**Comment by olafurpg** [comment]
Any idea why the CI is not triggering?

File: src/FileIndexer.ts [hunk 2]
```diff
@@ -14,27 +14,23 @@ import {
   typeParameterDescriptor,
 } from './Descriptor'
 import { Input } from './Input'
-import * as lsif from './lsif'
-import { LsifSymbol } from './LsifSymbol'
-import { lsiftyped } from './main'
 import { Packages } from './Packages'
 import { Range } from './Range'
+import * as scip from './scip'
+import { ScipSymbol } from './ScipSymbol'
```
Comment (varungandhi-src) [inline src/FileIndexer.ts:20]: nit: Following JS/TS conventions, I think this should be called SCIPSymbol, similar to HTML and URL in browser APIs.

**Review by varungandhi-src — APPROVED** [review]

File: src/FileIndexer.ts (hunk 2 above)
Comment (olafurpg) [reply src/FileIndexer.ts:20]: I recall seeing in relation to `XMLHttpRequest` that the convention is that only three-letter acronym should remain uppercase. I really dislike SCIPSymbol, it's easier IMO to read ScipSymbol so I prefer to keep it

**Comment by varungandhi-src** [comment]
Nothing seems to have changed in the CI config, so I'm as confused as you are. Some transient GitHub issue? Maybe try pushing a dummy commit?

**Comment by olafurpg** [comment]
Empty commit seems to have done the trick

**Pull request merged**